bark_core/signatures.py

```diff
@@ -385,6 +387,17 @@ def get_authorized_pubkeys(
 
 def get_pubkey_from_git() -> Optional[Pubkey]:
     identifier = cmd("git", "config", "user.signingKey", check=False)[0]
+    gpg_format = cmd("git", "config", "gpg.format", check=False)[0]
+    # check if signingKey is ssh
+    if gpg_format and gpg_format == "ssh":
```

Comment:
There's no need to parse and check `gpg_format` if `identifier` is `None`. I think it would be better to get `gpg_format` and perform the check here:  https://github.com/YubicoLabs/gitbark-core/pull/21/files#diff-398bcc9bee5bed4a56368526a8c80e787bbbc105bb6b064fd6d7339b54413a3fR401.